spec/models/review_spec.rb

```diff
@@ -142,4 +142,42 @@
       review.valid?.should be_false
     end
   end
-end
\ No newline at end of file
+
+  describe "#upcoming?" do
+    it "returns false if review date is blank" do
+      review = create(:review)
+      review.review_date = nil
+      review.upcoming?.should be_false
+    end
+
+    it "returns false if review date is not in next six months" do
+      review = create(:review, review_date: 12.months.from_now)
+      review.upcoming?.should be_false
+    end
+
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.